test/Project/Controller/ImportProjectControllerTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace CultuurNet\ProjectAanvraag\Project\Controller;
+
+use CultuurNet\ProjectAanvraag\Address;
+use CultuurNet\ProjectAanvraag\Core\Exception\MissingRequiredFieldsException;
+use CultuurNet\ProjectAanvraag\Coupon\CouponValidatorInterface;
+use CultuurNet\ProjectAanvraag\Entity\Project;
+use CultuurNet\ProjectAanvraag\Entity\ProjectInterface;
+use CultuurNet\ProjectAanvraag\Insightly\InsightlyClientInterface;
+use CultuurNet\ProjectAanvraag\Insightly\Item\Contact;
+use CultuurNet\ProjectAanvraag\Insightly\Item\Link;
+use CultuurNet\ProjectAanvraag\Insightly\Item\Organisation;
+use CultuurNet\ProjectAanvraag\Integrations\Insightly\InsightlyClient;
+use CultuurNet\ProjectAanvraag\Integrations\Insightly\PipelineStages;
+use CultuurNet\ProjectAanvraag\Project\Command\ActivateProject;
```

Comment:
I think there are a lot of unused namespaces here.

Reply:
Indeed, I had too much faith in `make ci`